YoCode/UIKeywords.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace YoCode
+{
+    public struct UIKeywords
+    {
+        public static string[] UNIT_KEYWORDS = { "miles", "kilometres", "mile", "kilometre", "kilometers", "kilometer", "km", "mi" };
```

Comment:
Keyword "mi" creates a bug in UICheck where it will add additional files to the modifed list that shouldn't be there (in this case it matches every word "submit" and marks it as a UI change)